lib/src/helpers/default_options_list.dart

```diff
@@ -1,24 +1,44 @@
 import 'package:flutter/material.dart';
-import 'package:flutter_sfsymbols/flutter_sfsymbols.dart';
 import 'package:help_ukraine_widget/help_ukraine_widget.dart';
+import 'package:help_ukraine_widget/src/helpers/emojis.dart';
+import 'package:help_ukraine_widget/src/theme/font_config.dart';
 
 ///
 const defaultOptionsList = [
   HelpOptionButton(
     title: 'Donate',
-    icon: SFSymbols.money_dollar,
-    iconSize: 20,
+    child: Text(
+      Emojis.moneyWithWings,
+      style: TextStyle(
+        fontFamily: FontConfig.family,
+        fontSize: 20.8,
+        fontWeight: FontWeight.w500,
+      ),
+    ),
   ),
   HelpOptionButton(
     title: 'Support Ukraine',
-    icon: SFSymbols.heart_fill,
-    iconSize: 19,
     iconColor: Colors.redAccent,
+    child: Text(
+      Emojis.heart,
+      style: TextStyle(
+        fontFamily: FontConfig.family,
+        fontSize: 20.8,
```

Comment:
What do you think?
```suggestion
        fontSize: FontConfig.title.fontSize,
```

Reply:
Would it be okay to just factor out common textStyle in this file, but leave fontSize as is?
Because it's used in just one other file besides this, and with other boldness.